coordinator/src/main/java/org/apache/uniffle/coordinator/QuotaManager.java

```diff
@@ -53,13 +53,14 @@ public class QuotaManager {
   private final Map<String, Integer> defaultUserApps = Maps.newConcurrentMap();
 
   public QuotaManager(CoordinatorConf conf) {
-    this.quotaFilePath = conf.get(CoordinatorConf.COORDINATOR_QUOTA_DEFAULT_PATH);;
+    this.quotaFilePath = conf.get(CoordinatorConf.COORDINATOR_QUOTA_DEFAULT_PATH);
```

Comment:
If `quotaFilePath` is empty, should we throw an exception here  right now?

Reply:
If the quotaFilePath is empty, the quota for each user is the default value of rss.coordinator.quota.default.app.num. No exception is thrown, but logs can be added

Reply:
I mean if quotaFilePath is empty, we should not try `getFilesystem` or create `ScheduledExecutorService`